src/main/java/dev/galacticraft/mod/api/block/FluidLoggable.java

```diff
@@ -137,7 +135,7 @@ default boolean isEmpty(BlockState state) {
         return state.getValue(FLUID).equals(Constant.Misc.EMPTY) || state.getValue(FLUID).equals(INVALID);
     }
 
-    //TODO Correct fluid pickup sound
+    //TODO Correct fluid pickup sound, probably unfixable unless mojang added "BlockState" argument into this method
```

Comment:
We can probably use a mixin here to fix this if we want it to work with vanilla atleast

Reply:
Done in https://github.com/TeamGalacticraft/Galacticraft/pull/264/commits/b9e1bc662b7aaa58ffbb6020b395e6571e4a4e92.